### ISSUE TITLE
question: why is tail at the end of entrypoint.sh?

### ISSUE THREAD

**Issue by kewalaka** [google/moodle-on-gcp#24]
Using tail -f /dev/null as the main process in a container seems inadvisable because tail doesn't respond to signals like SIGHUP, SIGINT, or SIGTERM in a meaningful way.

It also seems redundant, because the nginx -g 'daemon off;' command already keeps the container running in the foreground.   Without tail, Nginx would be acting as the main process in the container?

Would you be able to share the reason for this please?

**Comment by kewalaka** [comment]
ref - https://github.com/google/moodle-on-gcp/blob/main/4-moodle-image-builder/base/opt/entrypoint.sh#L24C1-L24C18

**Comment by ernani** [comment]
That's true, I will do some improvements in that part ASAP, or, you are welcome to submit a PR if you want. Thanks!

**Comment by ernani** [comment]
@kewalaka Fixed it.
